modules/stochastic_tools/doc/content/syntax/ParameterStudy/index.md

```diff
@@ -0,0 +1,158 @@
+
+## Outputs
+
+The `ParameterStudy` syntax provides ways to output the results of the study in CSV and/or JSON format. The sampler matrix (the values of the perturbed parameters) can be outputted by setting the [!param](/ParameterStudy/sampler_output_type) parameter to `csv` and/or `json`. The resutling QoI values can be outputted by setting the [!param](/ParameterStudy/qoi_output_type) to `csv` or `json`. However, the CSV output will not contain vector-type QoIs, like those from vector-postprocessors. The default for both of these parameters is `json`. A JSON output is automatically created if QoI statistics are computed.
+
+## List of Objects
+
+All the objects built with [ParameterStudyAction.md] can be shown on console using the [!param](/ParameterStudy/show_study_objects) parameter. [tab:study_objects] lists all the objects created.
+
+!table id=tab:study_objects caption=Objects created using the `ParameterStudy` syntax
+| Base Type | Type | Name |
+| - | - | - |
+| Action | [StochasticToolsAction.md] | `ParameterStudy_stochastic_tools_action` |
+| [Distribution](Distributions/index.md) | See [#sec:random] | `study_distribution_<i>`$^a$ |
+| [Sampler](Samplers/index.md) | See [#sec:sampling] | `study_sampler` |
+| [MultiApp](MultiApps/index.md) | [SamplerFullSolveMultiApp.md] | `study_app` |
+| [Control](Controls/index.md) | [MultiAppSamplerControl.md] | `study_multiapp_control`$^b$ |
+| [Transfer](Transfers/index.md) | [SamplerParameterTransfer.md] | `study_parameter_transfer`$^c$ |
+| [Transfer](Transfers/index.md) | [SamplerReporterTransfer.md] | `study_qoi_transfer`$^d$ |
+| [VectorPostprocessor](VectorPostprocessors/index.md) | [SamplerData.md] | `study_samples`$^e$ |
+| [Reporter](Reporters/index.md) | [StochasticReporter.md] | `study_qoi`$^d$ |
+| [Reporter](Reporters/index.md) | [StatisticsReporter.md] | `study_statistics`$^{d,f}$ |
+| [Output](Outputs/index.md) | [CSV.md] | `csv`$^g$ |
+| [Output](Outputs/index.md) | [JSONOutput.md] | `json`$^h$ |
+
+!style fontsize=small
+$^a$ `<i>` corresponds to the index of the distribution in [!param](/ParameterStudy/distributions);\\
+$^b$ If [!param](/ParameterStudy/multiapp_mode) is `normal` or `batch-reset`;\\
+$^c$ If [!param](/ParameterStudy/multiapp_mode) is `batch-restore`, `batch-keep-solution`, or `batch-no-restore`;\\
+$^d$ If [!param](/ParameterStudy/quantities_of_interest) is specified;\\
+$^e$ If [!param](/ParameterStudy/sampler_output_type) is not `none`;\\
+$^f$ If [!param](/ParameterStudy/compute_statistics) is `true`;\\
+$^g$ If [!param](/ParameterStudy/sampler_output_type) or [!param](/ParameterStudy/qoi_output_type) contains `csv`;\\
+$^h$ If [!param](/ParameterStudy/sampler_output_type) or [!param](/ParameterStudy/qoi_output_type) contains `csv` or [!param](/ParameterStudy/compute_statistics) is `true`;
+
```

Comment:
Maybe add `Typically for parameter study, users want only having one input block [ParameterStudy]. But STM does not prevent users from adding more objects by using other valid STM input syntax including various MOOSE syntax. This section gives advanced users more information on how to make other syntax work with [ParameterStudy] seamlessly.`

Reply:
Thank you for the suggestion. Added.